resolwe_bio/processes/slamdunk/slam_count.py

```diff
@@ -0,0 +1,121 @@
+"""Convert collapsed Slamdunk tcount data to the expressions data type."""
+import os
+
+from plumbum import TEE
+
+from resolwe.process import Cmd, DataField, FileField, JsonField, Process, StringField
+
+import pandas as pd
```

Comment:
I would put this in the line above `from plumbum import TEE`. The logic being "system" packages first and then addons such as plumbum, resolwe and pandas. I would also not have any spaces between these three package imports.